dss/storage/checkout/bundle.py

```diff
@@ -116,6 +120,70 @@ def verify_checkout(
     raise CheckoutError(f"status: {status}")
 
 
+def _is_checkout_valid(
+        replica: Replica,
+        bundle_uuid: str,
+        bundle_version: typing.Optional[str],
+) -> bool:
+    """
+    Validates the contents of a checkout bundle against its bundle manifest.
+    Ensures the checkout bundle is not near expiration to avoid serving a missing bundle.
+    :param replica: Cloud replica
+    :param bundle_uuid: Bundle UUID
+    :param bundle_version: Bundle version
+    :return: Is checkout bundle valid
```

Comment:
Consider more precise wording: "True if checkout bundle is valid, False otherwise".